src/element-observation.js

```diff
@@ -63,7 +63,7 @@ export class StyleObserver {
   _setProperty(style, value) {
     let priority = '';
 
-    if (value && value.indexOf('!important') !== -1) {
+    if (typeof value !== typeof undefined && typeof value.indexOf !== typeof undefined && value.indexOf('!important') !== -1) {
```

Comment:
It could still be null. I think we want something like this:

```
if (value !== null && value !== undefined && typeof value.indexOf === 'function' && value.indexOf('!important') !== -1)
```

Does that make sense? If so can you update the PR with that?